### PR TITLE
LibVT: Show application display name when hovered over, if available

### LINKED ISSUE #14416: LibVT: "Open ... with ..." should show application display name, if available
i.e. "Text Editor", not "TextEditor".

image

### PR DESCRIPTION
**Pull request by Wuzado**
Fixes #14416.

### CODE REVIEW

**Review by MacDue — COMMENTED** [review]
Some nitpicks

File: Userland/Libraries/LibVT/TerminalWidget.cpp [hunk 1]
```diff
@@ -837,8 +837,14 @@ void TerminalWidget::mousemove_event(GUI::MouseEvent& event)
...
                     set_tooltip(String::formatted("Execute {}", name));
-                else
-                    set_tooltip(String::formatted("Open {} with {}", name, LexicalPath::basename(handlers[0])));
+                else {
+                    auto af = Desktop::AppFile::get_for_app(LexicalPath::basename(handlers[0]));
+                    if (af->is_valid()) {
+                        set_tooltip(String::formatted("Open {} with {}", name, af->name()));
+                    } else {
+                        set_tooltip(String::formatted("Open {} with {}", name, LexicalPath::basename(handlers[0])));
+                    }
```
[the remark this answers is not in the document]
Comment (Wuzado) [reply Userland/Libraries/LibVT/TerminalWidget.cpp:846]: Really like the ternary operator here, didn't think about it. Thanks for the suggestion!

**Comment by Wuzado** [comment]
Thanks for the review! Gonna apply the changes in another commit whenever I have the time to do so.

**Comment by MacDue** [comment]
You shouldn't put these changes in a another commit, just amend (`git commit --amend`) your current commit, then force push (`git push -f`)

**Comment by Wuzado** [comment]
Alright. Assumed that the maintainers would prefer to squash the PR instead, but considering the changes are primarily formatting changes... Still getting used to the pull request workflow in Serenity.

**Pull request merged**